compass/landice/tests/ismip6_run_ais/ismip6_run_ais.cfg

```diff
@@ -0,0 +1,44 @@
+# config options for ismip6 antarctic ice sheet data set
+[ismip6_run_ais]
+
+# Resolution that will be set up.  Should be one of 08, 04 with leading 0.  The paths below must manually be set to be consistent
+mesh_res = 08
+
+# COMPASS load script path
+# This is user-specific.
+compass_load_path = /global/cscratch1/sd/hoffman2/AIS8km/compass/load_dev_compass_1.2.0-alpha.1_cori-haswell_gnu_mpt_albany.sh
+
+# Base path to the processed input ismip6 ocean and smb forcing files. User has to supply.
+forcing_basepath = /global/project/projectdirs/piscees/MALI_projects/ISMIP6-2300/forcing/ais_mesh_8to30km_res
+
+# Path to the initial condition file. User has to supply.
+init_cond_path = /global/project/projectdirs/piscees/MALI_projects/ISMIP6-2300/initial_conditions/AIS_8to30km_20221027/relaxation_0TGmelt_10yr_muCap/AIS_8to30km_r01_20220906.smooth3.basinsFineTuned_carvedRonne_CIRWIP_relaxation_0TGmelt_10yr_muCap.nc
+
+# Path to the file for the basal melt parametrization coefficients.
+melt_params_path = /global/project/projectdirs/piscees/MALI_projects/ISMIP6-2300/initial_conditions/AIS_8to30km_20221027/basin_and_coeff_gamma0_DeltaT_quadratic_non_local.nc
+
+# Path to the region mask file
+region_mask_path = /global/project/projectdirs/piscees/MALI_projects/ISMIP6-2300/initial_conditions/AIS_8to30km_20221027/AIS_8to30km_r01_20220607.regionMask_ismip6.nc
+
+# Path to the directory containing graph files
+graph_files_path = /global/project/projectdirs/piscees/MALI_projects/ISMIP6-2300/initial_conditions/AIS_8to30km_20221027
```

Comment:
Can these be generated by this test case rather than being supplied?

Reply:
Good question.  I had two issues when I tried to do that:
1. `make_graph_file` seemed to not do anything when called during `configure`.  Because I am not intending for `compass run` to be used to actually run the model, it might not be possible to use `make_graph_file`
2. Because we are intending for a user to use the slurm batch script outside of `compass run`, we also need the `graph.info.part.*` file for the number of processors used.  These are meant to be pre-generated for the desired number of processors the experiments should be run with.  

So we either need flexibility to use the graph file generation tools in a nonstandard way, or I need to be convinced that using compass run to run the experiments is a good idea.  (Or we leave some of the nonstandard steps to users to supply themselves, given the atypical nature of this test case.)